main/assets/sass/base/_typography.scss

```diff
@@ -2,4 +2,10 @@
 
 address {
     font-style:normal;
+}
+
+.text-truncate {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
 }
```

Comment:
text truncate need more clear

Reply:
// Text truncate
// Requires inline-block or block for proper styling

@mixin text-truncate() {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}


.text-truncate {
    @include text-truncate;
}